h2/src/docsrc/help/help.csv

```diff
@@ -4852,6 +4852,23 @@ This method returns a string.
 CALL STRINGENCODE(STRINGDECODE('Lines 1\nLine 2'))
 "
 
+"Functions (String)","BASE64_ENCODE","
+@h2@ BASE64_ENCODE(bytes[, algorithmString])
+","
+Encodes a byte array in the Base64 format to a string.
```

Comment:
The source byte array is not in the Base64 format, the result of this function is.

Also describe both possible values of `algorithmString` and which one is used by default.